example/states/webview/src/views.tsx

```diff
@@ -39,8 +37,8 @@ export class PolylineArrowEdgeView extends PolylineEdgeView {
 }
 
 @injectable()
-export class TriangleButtonView implements IView {
-    render(model: SPort, context: RenderingContext, args?: object): VNode {
+export class TriangleButtonView<A extends IViewArgs> implements IView<A> {
+    render(model: SPort, context: RenderingContext, args?: A): VNode {
```

Comment:
Please remove the `args` (they are not used) and the type parameter.

Reply:
I did not know this was possible in TypeScript to just leave out the parameter just like that. Couldn't the other two parameters be left out for the same reasoning as well because they are also not used?

Reply:
Yes, you can leave out any parameter that you don't use as long as you maintain the order. In this case I would leave the first two because it keeps the important parts of the API visible. The third `args` parameter is a rather special case that is not required in most user code.